stonesoup/models/measurement/nonlinear.py

```diff
@@ -138,7 +138,7 @@ def _rotation_matrix(self) -> np.ndarray:
         """
 
         theta_x = -self.rotation_offset[0, 0]
-        theta_y = -self.rotation_offset[1, 0]
+        theta_y = self.rotation_offset[1, 0]
         theta_z = -self.rotation_offset[2, 0]
 
```

Comment:
I think it would be better if MovingPlatform's orientation was changed instead of the rotation matrix. That way we can keep the rotations as clockwise.

Reply:
That wouldn't fix these tests: they only uses models, not platforms.

The velocity->orientation transform in my tests is done (as it is in platform, but it looks sensible to me) using.
``` python
    _, bearing, elevation = cart2sphere(*sensor_velocity)
    orientation = StateVector([0, elevation, bearing])
```

How would you change this?

Reply:
I've given it some more thought and I think It will be fine as `theta_y = self.rotation_offset[1, 0]` but it is probably a good idea to specify how the rotations work in the doc string and checking if we need to change any other rotation matrices.